2022-01-20 17:51:00,291 CRITICAL client_log_config.py Критическая ошибка
2022-01-20 17:51:00,292 ERROR client_log_config.py Обнаружена ошибка
2022-01-20 17:51:00,292 DEBUG client_log_config.py Отладочная информация
2022-01-20 17:51:00,292 INFO client_log_config.py Информационное сообщение
2022-01-23 15:42:41,177 INFO client.py Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 7777
2022-01-23 15:42:43,242 CRITICAL client.py Не удалось подключиться к серверу 127.0.0.1:7777, удаленный компьютер отказал в запросе на подключение.
2022-01-23 15:50:22,561 INFO client.py Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 7777
2022-01-23 15:50:24,611 CRITICAL client.py Не удалось подключиться к серверу 127.0.0.1:7777, удаленный компьютер отказал в запросе на подключение.
2022-01-23 15:51:33,512 INFO client.py Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 7777
2022-01-23 15:51:35,566 CRITICAL client.py Не удалось подключиться к серверу 127.0.0.1:7777, удаленный компьютер отказал в запросе на подключение.
2022-01-23 16:31:43,018 INFO client.py Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 7777
2022-01-23 16:31:43,019 DEBUG client.py Сформировано presence сообщение для пользователя Client
2022-01-23 16:31:43,022 DEBUG client.py Разбор сообщения от сервера: {'response': 200}
2022-01-23 16:31:43,022 INFO client.py Принят ответ от сервера The response is "200 : Ok"
